core/server/master/src/main/java/alluxio/master/backup/AbstractBackupRole.java

```diff
@@ -186,9 +186,9 @@ protected AlluxioURI takeBackup(BackupPRequest request, AtomicLong entryCounter)
       } catch (IOException e) {
         try {
           ufs.deleteExistingFile(backupFilePath);
```

Comment:
What is the difference between `backupFilePath` and `backupUri`?

Reply:
`backupFilePath` is the given path+ generated file name.
`backupUri` is actual location that backup will be written based on other options. (i.e --local)